public/locales/en-US/send.ftl

```diff
@@ -117,3 +126,30 @@ passwordIsSet = Password set
 maxPasswordLength = Maximum password length: { $length }
 # A short status message shown when there was an error setting the password
 passwordSetError = This password could not be set
+pageHeaderCredits = from the makers of Firefox
+addFilesButton = Add file(s)
+uploadFilesButton = Send
+uploadFileProgress = Sending
+uploadDropDragMessage = Drop files here
+uploadDropClickMessage = or click to select a file
+addPasswordMessage = Protect with password
+addPasswordLabel = Password:
+copyUrlLabel = Copy and share this link:
+passwordReminder = don't forget the password too
+signInPromoText = Sign In/Up!
+signInExplanation = It's free and gives you many more Send options
+signInLearnMore = Learn more!
+downloadProgressButton = Downloading... { $progress }
```

Comment:
Always use single unicode character for ellipsis: `…` vs `...`. Also, when adding a variable, it helps to add a comment explaining what's going to replace it with an example.